packages/angular/src/builders/webpack-browser/webpack-browser.impl.ts

```diff
@@ -69,12 +88,41 @@ function run(
     dependencies
   );
 
+  // If we don't have a third-party builder being used
+  // And there is a path to custom webpack config
+  // Invoke our own support for custom webpack config
+  let runBuildWithCustomWebpackConfig = { shouldRun: false, pathToConfig: '' };
```

Comment:
I think it would be better to move the logic to decide how to build the app to the `buildApp` function. We are already making decisions in that function about how to do it. By adding this here, we now have two places making decisions on how to build the app.

Reply:
The main problem I see with that is that the return types of `executeBrowserBuilder` and `scheduleBuilder` don't match, and therefore have to be piped differently.

I guess I could do a little rework that would make that happen though :)

Reply:
Yep, I think it's fine. `buildApp` can standardize what gets returned.